libs/components/src/lib/card/card.ts

```diff
@@ -71,6 +72,89 @@ export class Card extends VividElement {
 	 */
 	@attr elevation?: 0 | 2 | 4 | 8 | 12 | 16 | 24;
 
+	/**
+	 * Indicates whether card should be a clickable <button> element.
+	 *
+	 * @public
+	 */
+	@attr({
+		mode: 'boolean',
+		attribute: 'clickable-card',
+	})
+	clickableCard = false;
+
+	/**
+	 * Type of a <button> element.
+	 *
+	 * @public
+	 * @remarks
+	 * HTML Attribute: type
+	 */
+	@attr type?: string;
+
+	/**
+	 * Indicates the button's href.
```

Comment:
```suggestion
	 * Indicates the card's href.
```

But tbh these descriptions provide no value anyway so they might as well be removed enitrely